Rakefile.rb

```diff
@@ -22,7 +22,7 @@
 
 desc "Build the site"
 task :build => :check do
-  system "bundle exec awestruct -P production --force"
+  system ("bundle exec awestruct -P production --force") or raise "An error occured during the build. Please see the log for details"
```

Comment:
There still a space before "(", which is odd, and may cause things to break.
```suggestion
  system("bundle exec awestruct -P production --force") or raise "An error occured during the build. Please see the log for details"
```

Reply:
Fixed